docs/how_to/apphooks.rst

```diff
@@ -60,6 +60,8 @@ then your ``MyApphook`` class should include::
 If you fail to this, then any templates in the application that invoke URLs using the form ``{% url 'myapp:index' %}``
 or views that call (for example) ``reverse('myapp:index')`` will throw a ``NoReverseMatch`` error.
 
+If you had already assigned a page to your aplication prior to including ``app_name``, you'll need to edit its *Advanced settings* and specify your  ``app_name`` as *Application instance name* to avoid the same ``NoReverseMatch`` error (this is filled automatically in new pages).
```

Comment:
I suggest "prior to setting the `app_name` attribute"
Also "to avoid a ``NoReverseMatch`` error